src/quick-lint-js/fe/parse-expression.cpp

```diff
@@ -2610,7 +2615,8 @@ Expression* Parser::parse_arrow_function_expression_remainder(
 }
 
 Expression::Call* Parser::parse_call_expression_remainder(Parse_Visitor_Base& v,
-                                                          Expression* callee) {
+                                                          Expression* callee,
+                                                          std::optional<Source_Code_Span> optional_chaining_op_) {
```

Comment:
Nit: Parameters and local variables should not have a trailing underscore:
```suggestion
                                                          std::optional<Source_Code_Span> optional_chaining_op) {
```